### DISCUSSION TITLE
SocketException: Connection refused (OS Error: Connection refused, errno = 61), address = localhost, port = 56388

### DISCUSSION THREAD

**Discussion by qcks** [ra1u/redis-dart#70] [Q&A]
no work

**Comment by ra1u** [comment]
It seems that your redis server is not available on this port and host.
Are you able to connect on redis server without use of this library?

If your server is properly configured, you should be able to connect on redis using [redis-cli](https://redis.io/docs/manual/cli/) 

    redis-cli -h localhost -p 56388 PING

**Reply by silicon-ninja** [reply]
isn't the default port 6379 ? i was trying to run my app in release mode and it breaks saying couldn't connect.
